spec/commands/sscan_spec.rb

```diff
@@ -7,12 +7,12 @@
 
   before do
     # The return order of the members is non-deterministic, so we sort them to compare
-    expect_any_instance_of(Redis).to receive(:sscan).and_wrap_original do |m, *args|
-      result = m.call(*args)
+    expect_any_instance_of(Redis).to receive(:sscan).and_wrap_original do |m, *args, **kwargs|
+      result = m.call(*args, **kwargs)
       [result[0], result[1].sort]
     end
-    expect_any_instance_of(MockRedis).to receive(:sscan).and_wrap_original do |m, *args|
-      result = m.call(*args)
+    expect_any_instance_of(MockRedis).to receive(:sscan).and_wrap_original do |m, *args, **kwargs|
+      result = m.call(*args, **kwargs)
```

Comment:
Resolve issues running the test suite on Ruby 3.2.

There have been some fixes to the separation of method arguments and keyword arguments in this version of Ruby.

Resources:
- [Separation of positional and keyword arguments in Ruby](https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/)
- [Ruby 3.2.0 Released](https://www.ruby-lang.org/en/news/2022/12/25/ruby-3-2-0-released/)